fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -116,6 +127,11 @@ def gamma(self, predictor):
                                                     _LOWER_BOUND_DIFF]])
         return g_signed
 
+    def bound(self):
+        if self.eps is None:
+            raise ValueError("No Bound")
```

Comment:
Perhaps this should throw `NotImplementedError` if `self.eps is None`? @MiroDudik

Reply:
Ah, you had that in the child classes :-)
`eps` should really be set in the constructor of the classes that need it. Otherwise I think we can throw `NotImplementedError`

Reply:
So just ot make sure, should we throw NotImplementedError instead of ValueError here if self.eps is None?

Reply:
That's fine. I'll close this one and we can reopen it or start a new thread if we need more revisions.